qa/L0_backend_python/model_control/test.sh

```diff
@@ -37,9 +37,12 @@ source ../../common/util.sh
 
 mkdir -p models/identity_fp32/1/
 mkdir -p models/simple_identity_fp32/1/
+mkdir -p models/faulty_model/
```

Comment:
For faulty model, I think we could reuse this one: https://github.com/triton-inference-server/server/blob/main/qa/python_models/auto_complete_error/model.py